docs/pages/validator-management/vc-configuration.md

```diff
@@ -88,11 +88,29 @@ Configure your validator client's fee recipient address by using the `--suggeste
 
 You may choose to use the `--strictFeeRecipientCheck` flag to enable a strict check of the fee recipient address with the one returned by the beacon node for added reassurance.
 
+### Configure your builder selection and/or builder boost factor
+
+If you are running a Lodestar beacon with connected builder relays, you may use these validator configurations to signal which block (builder vs. local execution) for the beacon node to propose. 
```

Comment:
Technically, the behavior should be the same for all beacon nodes, the only difference (non spec'd behavior) to other clients are:
- we reject local (`builderonly` ) and builder block (`executiononly`) on vc side
- we do not trigger builder (`executiononly`) and local (`builderonly`) block production on the bn side

```suggestion
If you are running a beacon node with connected builder relays, you may use these validator configurations to signal which block (builder vs. local execution) the beacon node should produce.
```